labeled_data/bot/index.yml

```diff
@@ -0,0 +1,13 @@
+name: GitHub Bot
+type: Bot
+data:
+  label:
+    - github_bot
```

Comment:
You should delete these 2 lines, there is no sub label called `github_bot`

Reply:
So, do I needn't add a label? And the sub key uses `github_user` representing GitHub account is OK?